azurerm/internal/services/appplatform/spring_cloud_service_resource.go

```diff
@@ -422,6 +480,17 @@ func expandArmSpringCloudGitPatternRepository(input []interface{}) (*[]appplatfo
 	return &results, nil
 }
 
+func expandArmSpringCloudTrace(input []interface{}) *appplatform.TraceProperties {
+	if len(input) == 0 || input[0] == nil {
+		return nil
```

Comment:
(as above) we can take the omission of this block to mean this is Disabled - so we can make this:

```suggestion
		return &appplatform.TraceProperties{
			Enabled: utils.Bool(false),
		}
```